react-ui/src/components/Layouts/Profile.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component, Fragment } from "react";
 import Title from "../Title"
 
-class Page2 extends Component {
```

Comment:
This file was renamed to increase clarity and avoid user confusion (as mentioned in the TA demo) but no actual work was done for this page for this PR (out of scope).

Reply:
Yes, I agree - thank you for renaming this file, will definitely help avoid any confusion.